doc/conf.py

```diff
@@ -16,6 +16,8 @@
 
 html_theme = 'sphinx_rtd_theme'
 
+execution_timeout = 90
+
```

Comment:
Default value is 30. Under certain circumstances, I guess it takes longer than 30 s to download the echemdb database, and then the build process is terminated.